tests/jerry/es2015/typedArray-find.js

```diff
@@ -0,0 +1,116 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var arrow_is_available = false;
+
+try {
+  assert (eval ("(f => 5) ()") === 5);
+  arrow_is_available = true;
+} catch (e) {
+  assert (e instanceof SyntaxError);
+}
+
+var typedArrayTypes = [Int8Array,
+                       Uint8Array,
+                       Uint8ClampedArray,
+                       Int16Array,
+                       Uint16Array,
+                       Int32Array,
+                       Uint32Array,
+                       Float32Array,
+                       Float64Array];
+
+
+typedArrayTypes.forEach (function (TypedArray) {
+
+  var array1 = new TypedArray ([5, 12, 0, 8, 120, 44]);
+
+  function bigger_than_10 (element) {
+    return element > 10;
+  }
+
+  assert (array1.find (bigger_than_10) === 12);
+
+  function less_than_0 (element) {
+    if (element == 0) {
+      throw new Error ("zero");
+    }
+    return element < 0;
+  }
+
+  try {
+    array1.find (less_than_0);
+    assert (false);
+  } catch (e) {
+    assert (e instanceof Error);
+    assert (e.message === "zero");
+  }
+
+  if (arrow_is_available) {
+      assert((array1.find (e => e > 100)) === 120);
```

Comment:
Is this really correct? the `arrow_is_available` will be calculated run-time, but the `=>` here is a syntax "sugar" which is processed when parsing the source. AFAIK: this should also be in an "eval" function.

Reply:
Yes, you are right. Anyway, I used `eval` in the other test file, but it seems it left out here.